lib/FinderNavigation/styles.scss

```diff
@@ -4,187 +4,123 @@
-  padding-left: $layout-spacing-base/4;
-  color: $neutral-base;
-  .button,
-  .dropdown-gc {
-    vertical-align: text-bottom;
-    color: $neutral-dark;
+.finder-item-active > .finder-item-content {
+  @apply bg-white text-blue-primary shadow-small;
+  &:hover {
+    @apply bg-white;
   }
-  .button + .dropdown-gc {
-    margin-left: $layout-spacing-base/2;
+  .finder-item-link {
+    @apply text-blue-primary;
+    .icon path {
+      fill: theme('colors.blue.primary');
+    }
   }
 }
 
-.finder__settings-dropdown {
-  right: -$layout-spacing-base/2;
-  line-height: 20px;
-  &.is-active {
-    top: $layout-spacing-base + $layout-spacing-base/4;
-  }
+.finder-item-content-hidden {
+  @apply h-0 overflow-hidden;
 }
 
-.finder__item-link {
-  @include text-overflow-ellipsis();
-  flex-shrink: 1;
-  cursor: pointer;
+.finder-item-content-inner {
+  @apply flex w-full h-full items-center;
 
-  &:focus {
-    outline: none;
+  .form__input {
+    @apply py-0 pl-3 pr-0;
   }
 }
 
-.finder__item-no-indentation,
-.finder__item {
-  color: $neutral-dark;
-  font-size: $typo-size-base;
-
-  &-link {
-    color: $neutral-dark;
-    text-decoration: none;
-    display: flex;
-    align-items: center;
-    &:hover {
-      color: $primary-blue;
-    }
+.finder-item-settings {
+  @apply ml-auto flex-shrink-0 pl-1 flex items-center text-neutral-primary;
+  .dropdown__content {
+    @apply mt-0 -mr-2 text-neutral-20;
   }
+}
 
-  .finder__group > div:last-child > .finder__item {
-    margin-bottom: 0;
+.finder-item-hover-settings {
+  .finder-item-settings {
+    display:none;
   }
-
-  &-link-icon {
-    margin-right: $layout-spacing-base/2;
-    height: $icon-height;
-    width: $icon-width;
-    display: flex;
+  &:hover {
+    .finder-item-settings {
+      @apply flex;
+    }
   }
 }
-.finder__item-no-indentation {
-  margin: 0;
+
+.finder-item-link {
+  @apply no-underline text-neutral-20 flex-shrink w-full flex items-center cursor-pointer outline-none h-full truncate;
+  .icon {
+    @apply mr-3;
+  }
 }
 
 .finder__separator {
+  @apply bg-neutral-90 text-neutral-90 border-0 m-3;
   height: 2px;
-  color: $neutral-light;
-  background-color: $neutral-light;
-  border: none;
-  margin: $layout-spacing-base/2;
 }
 
 /* ==========================================================================
-   Modifiers
+   Drag and drop
    ========================================================================== */
 
-.finder__item.is-active > .finder__item-content {
-  background: white;
-  border-radius: $layout-spacing-base/4;
-
-  .finder__item-link {
-    color: $primary-blue;
-  }
-}
-
-.finder__item--hover-settings .finder__item-content .finder__item-settings {
-  display: none;
-}
-
-.finder__item--hover-settings
-  .finder__item-content:hover
-  .finder__item-settings {
-  display: flex;
-}
-
-.finder__item.is-over.dragged-insert > .finder__item-content,
-.finder__item--selected > .finder__item-content {
+ .finder-item-content--drag-wrapper {
```

Comment:
dnd and indentation stuff is pretty fiddly and specific, so I've swerved it for now!